### ISSUE TITLE
Support CycloneDX 1.5

### ISSUE THREAD

**Issue by candrews** [interlynk-io/sbomasm#18]
The CycloneDX specification was updated to version 1.5: https://cyclonedx.org/news/cyclonedx-v1.5-released/

And tools are already updating to support it, such as Trivy with its 0.43.0 release: https://github.com/aquasecurity/trivy/releases/tag/v0.43.0

Can this tool please also support the latest version of the CycloneDX specification?

**Comment by riteshnoronha** [comment]
@candrews yes we are currently evaluating 1.5 and will be updating all our tools to support Cyclonedx 1.5 and SPDX 3.0. Will update this thread once we get this done.

**Comment by riteshnoronha** [comment]
@candrews 1.6 support has been added as of 0.1.1
